clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -0,0 +1,83 @@
+package com.linkedin.venice.datarecovery;
+
+import com.linkedin.venice.controllerapi.ControllerClient;
+import com.linkedin.venice.controllerapi.MultiStoreInfoResponse;
+import com.linkedin.venice.meta.Store;
+import com.linkedin.venice.meta.StoreInfo;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.compress.utils.Sets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+public class DataRecoveryClient {
+  private static final Logger LOGGER = LogManager.getLogger(DataRecoveryClient.class);
+  private final static String STORE_NAME_SEPARATOR = "[,:]+";
+  private final DataRecoveryModule executor;
```

Comment:
My original thought is that `DataRecoveryClient` would include three modules: a `planner`, an `executor`, and a `monitor` in future and all three modules can share a common base class to handle the multithreading part of work. Sure, I can do the renaming.

Reply:
Emm.. 
Do we need multi-threading support for planning and monitoring?

Reply:
Do you mean that the number of stores in total is not too large, so that we can just query them in sequence? In that case, we can config the `poolSize` to be 1 for that module. Still want to leave it as one option for us, in case we see a need such parallelism later (e.g. api is slow in responding or something other case).

Reply:
Sure, either way is ok.
Controller requests are still much faster compared with the actual recovery, which may take hours :)